### PR TITLE
Webhost: Fix a typo on Start Playing page

### PR DESCRIPTION
**Pull request by ScipioWright**
## What is this fixing or adding?
On the start playing page, it says "You'll need to upload either a config file or a zip file containing one more config files."
Instead of "one more", it should be "one or more".

## How was this tested?
Standardized common core

## If this makes graphical changes, please attach screenshots.
N/A

### CODE REVIEW

**Review by Ixrec — APPROVED** [review]
I could see it being argued both ways since this text is aimed at people who are just learning AP for the first time.

Either way, the lack of "or" here is a pretty clear mistake we should fix.

**Comment by ScipioWright** [comment]
> Out of scope perhaps, but since you're in here, two things:
> 
>     1. Is "config file" a term we use for Player Option YAMLs anywhere else? It seems like an outdated term.
> 
>     2. The verbage here doesn't really capture the possibility of uploading multiple YAMLs together, not in a zip archive.

For the sake of clarity, when uploading yamls to the site to generate a game, can you only upload player option yamls, or can you upload stuff like meta.yaml, host.yaml, or others I'm not thinking of too?

**Comment by ScipioWright** [comment]
> > I could see "config file" being argued both ways since this text is aimed at people who are just learning AP for the first time, and jumping straight into Archipelago-specific jargon could be more intimidating than helpful.
> 
> I see where you're coming from, but I don't know that I see "config file" as so much more generic of a term that it's worth the subsequent confusion when they go any further than this specific page and never see the term again.

I'll probably just do something along the lines of "a yaml file (player options configuration file)" to make it a little clearer to a first-time player.

**Comment by LegendaryLinux** [comment]
I chose to use config as a generic term because something I would truly love is for normal users to never need to know what a YAML file is. Modifying them manually is in my opinion a power-user choice, or if it is not, it should be. Either way, I do not believe anyone is confused by the usage of the word "config."

I do agree the wording could be updated to reflect the possibility of uploading "one or more config files, or a single zip file containing one or more config files."

File: WebHostLib/templates/startPlaying.html [hunk 1]
```diff
@@ -18,7 +18,7 @@ <h1>Start Playing</h1>
                 <br /><br />
 
                 To start playing a game, you'll first need to <a href="/generate">generate a randomized game</a>.
-                You'll need to upload either a config file or a zip file containing one more config files.
+                You'll need to upload one or more config files (YAMLs) or a zip file containing one or more config files.
```
Comment (ScipioWright) [inline WebHostLib/templates/startPlaying.html:21]: I just committed a change here. Thoughts?

**Review by LegendaryLinux — APPROVED** [review]

**Pull request merged**